src/structures/ClientApplication.js

```diff
@@ -67,9 +68,9 @@ class ClientApplication extends Base {
 
     /**
      * The owner of this OAuth application
-     * @type {?User}
+     * @type {User|Team}
```

Comment:
The Owner and Team should be separated in my opinion. While the owner of an app that is in a team is fake, it's still an user. A warning should be added instead